build.sbt

```diff
@@ -93,6 +93,7 @@ lazy val commonSettings = Seq(
   //testOptions in Test  += Tests.Argument("-oD"),
   fork in Test         := true,
   resolvers            += Resolver.sonatypeRepo("public"),
+  resolvers            += Resolver.mavenLocal,
```

Comment:
I think we either need to figure out how to have gradle publish locally to .ivy in htsjdk, or leave this in if we want to be able to use locally published htsjdk artifacts.